### ISSUE TITLE
bug: 4-bit quantization not working with llama-2

### ISSUE THREAD

**Issue by crclark** [bentoml/OpenLLM#198]
### Describe the bug

Crashes on startup with:

```
Loading checkpoint shards: 100%|██████████| 2/2 [00:05<00:00,  2.77s/it]
2023-08-10T21:36:23+0000 [ERROR] [cli:llm-llama-runner:1] An exception occurred while instantiating runner 'llm-llama-runner', see details below:
2023-08-10T21:36:23+0000 [ERROR] [cli:llm-llama-runner:1] Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/openllm/_llm.py", line 845, in model
    model = model.to("cuda")
  File "/usr/local/lib/python3.10/dist-packages/transformers/modeling_utils.py", line 1895, in to
    raise ValueError(
ValueError: `.to` is not supported for `4-bit` or `8-bit` models. Please use the model as it is, since the model has already been set to the correct devices and casted to the correct `dtype`.

```

### To reproduce

```Dockerfile
FROM nvcr.io/nvidia/pytorch:23.07-py3
RUN pip install openllm[llama]==0.2.20 protobuf==3.20.2
```

```bash
docker build . -t openllm

docker run --gpus all --ipc=host --ulimit memlock=-1 --ulimit stack=67108864 --rm -p8000:3000 openllm openllm start llama --model-id NousResearch/llama-2-7b-hf --quantize int4
```

### Logs

```shell
see above
```

### Environment

bentoml 1.1.1

- `transformers` version: 4.31.0
- Platform: Linux-5.10.0-23-cloud-amd64-x86_64-with-glibc2.31
- Python version: 3.9.2
- Huggingface_hub version: 0.16.4
- Safetensors version: 0.3.2
- Accelerate version: 0.21.0
- Accelerate config:    not found
- PyTorch version (GPU?): 2.0.1+cu117 (True)
- Tensorflow version (GPU?): not installed (NA)
- Flax version (CPU?/GPU?/TPU?): not installed (NA)
- Jax version: not installed
- JaxLib version: not installed
- Using GPU in script?: <fill in>
- Using distributed or parallel set-up in script?: <fill in>

**Comment by aarnphm** [comment]
This has been fixed upstream, I'm in the progress of releasing a patch

**Comment by aarnphm** [comment]
This has been fixed with 0.2.22

**Comment by crclark** [comment]
Verified 0.2.22 is fixed; thanks!
